### DISCUSSION TITLE
oVirt 4.5.5 compatibility with my CPU

### DISCUSSION THREAD

**Discussion by ejakoje** [oVirt/ovirt-site#3178] [Q&A] [answered]
Hi all,

I'm planning to install ovirt node 4.5.5. on 2 new servers but I am not sure if this is possible since mine
servers have CPU "Intel Xeon Silver 4410Y 12C 150W 2.0GHz Processor" which is not supported according to this document: https://www.ovirt.org/documentation/installing_ovirt_as_a_self-hosted_engine_using_the_command_line/index.html#host-requirements
Does anyone know the answer?
Thank you

**Comment by sandrobonazzola** [comment, marked as answer]
Looking at https://www.redhat.com/en/blog/red-hat-enterprise-linux-achieves-significant-performance-gains-intels-4th-generation-xeon-scalable-processors I think it will work.
